tox.ini

```diff
@@ -82,7 +82,7 @@ envlist =
     mako_contrib-{py27,py34,py35,py36}-mako{010,100}
     molten_contrib-{py36}-molten{070,072}
     mongoengine_contrib-{py27,py34,py35,py36}-mongoengine{015}
-    msgpack_contrib-{py27,py34}-msgpack{03,04,05}
+    msgpack_contrib-{py27,py34}
```

Comment:
I'm confused; if we don't install any specific version of `msgpack`, which version it is tested against?
Are we removing support for 0.3, 0.4 and 0.5?

Reply:
This env/suite just tests our encoders (we have a JSON and msgpack encoder). So we are moving from testing the various versions the user could have installed to just our own internal one.

Let me rename this suite... I should also check if it is being run with the generic tracer suite or not, if so we can remove entirely.

Reply:
I agree the name is really confusing. :)

Reply:
I just removed this test env entirely since these tests get run by another anyways.